runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPanel.java

```diff
@@ -536,12 +553,38 @@ private void updateOverall()
 		}
 
 		overallKillsLabel.setText(htmlLabel("Total count: ", overallKills));
-		overallGpLabel.setText(htmlLabel("Total value: ", overallGp));
+
+		switch (plugin.getValueType())
+		{
+			case GRAND_EXCHANGE:
+				overallGeValueLabel.setText(htmlLabel("Total value: ", overallGeValue));
+				break;
+			case HIGH_ALCHEMY:
+				overallHaValueLabel.setText(htmlLabel("Total value: ", overallHaValue));
+				break;
+			case BOTH:
+				overallGeValueLabel.setText(htmlLabel("Total GE value: ", overallGeValue));
+				overallHaValueLabel.setText(htmlLabel("Total HA value: ", overallHaValue));
+		}
 	}
 
 	private static String htmlLabel(String key, long value)
 	{
 		final String valueStr = StackFormatter.quantityToStackSize(value);
 		return String.format(HTML_LABEL_TEMPLATE, ColorUtil.toHexColor(ColorScheme.LIGHT_GRAY_COLOR), key, valueStr);
 	}
+
+	void setValueType(LootTrackerValueType type)
+	{
+		overallGeValueLabel.setVisible(
+			EnumSet.of(LootTrackerValueType.GRAND_EXCHANGE, LootTrackerValueType.BOTH).contains(type));
```

Comment:
Just compare the enums directly and maybe abstract this logic. You have copy and pasted this a bit.

Reply:
de0366819507da1b1d7aa15cc39f99176f5f9c62